examples/multiple-scenario.js

```diff
@@ -0,0 +1,49 @@
+import { chromium } from 'k6/x/browser'
+
+export const options = {
+  scenarios: {
+    messages: {
+      executor: 'constant-vus',
+      exec: 'messages',
+      vus: 2,
+      duration: '2s',
+    },
+    news: {
+      executor: 'per-vu-iterations',
+      exec: 'news',
+      vus: 2,
+      iterations: 4,
+      maxDuration: '5s',
+    },
+  },
+  thresholds: {
+    browser_dom_content_loaded: ['p(90) < 1000'],
+    browser_first_contentful_paint: ['max < 1000']
+  }
+}
+
+export function messages() {
+  const browser = chromium.launch({
+    headless: __ENV.XK6_HEADLESS ? true : false,
+  });
+  const context = browser.newContext();
```

Comment:
In your WOLT presentation you were suggesting that there was an unnecessary step (possibly `const context = browser.newContext();`). So would it make is easier on new users if this was `const page = browser.newPage();`?

Reply:
You're right! I'll update it.